### ISSUE TITLE
404 after i type in the no gg thing

### ISSUE THREAD

**Issue by arseneangel** [ricochhet/Erupe#3]
![Screenshot 2022-01-06 191359](https://user-images.githubusercontent.com/97197315/148470407-7bec8645-9575-476a-8877-331ba9db10a9.png)

i messed around with things for like 4 hours with a friend who actually understands coding and we cannot get past this.

**Comment by Stefan240** [comment]
Well the 404 indicates the launcher cannot reach the server. 
Two things to check here at first: 

Make sure the server is actually running and not erroring out after being started. Powershell (or cmd) will show you a log after the server component is started with `go run .`

If you get something strange while starting the server you may copy the output here in a codeblock, just make sure to omit any personal info. 

After that make sure you edited your hosts-file correctly according to here: https://github.com/ricochhet/Erupe#step-4-edit-your-hosts-file

Also did you run any other MHF-Server before setting up this one?

**Comment by Synthiss** [comment]
I also have this issue. I've edited the hosts file accordingly and erupe was running when I launching.

**Comment by Stefan240** [comment]
@Synthiss anything in the logs?

**Comment by bimash** [comment]
I think the problem came from the hosts file because your launcher trying to connect to the MHF server and you probably have not delist the said domain

**Comment by Synthiss** [comment]
![Screenshot (3) - Copy](https://user-images.githubusercontent.com/97289178/148540959-597bc069-ea43-42a3-b250-244a5f39393e.png)


Here's what i did

**Comment by Stefan240** [comment]
@Synthiss Is this the first MHF-Server you ever connected to on this machine?
You may try this since sometimes the launcher can act funny: 

Sometimes the Monster Hunter Frontier Client can act funky when testing things and reinstalling or changing the server you play on. As a first step when troubleshooting things in the MHF-Client it can help to delete *Internet Explorers* (not Edge) Cache. This is pretty easy.
- Open up Internet Explorer
- Click on the little cogwheel on the top right
- Select Safety -> Delete browsing history
- In the now opening window check all the boxes and hit "Delete".

**Comment by Synthiss** [comment]
I see. It works now. Thanks for the help

**Comment by Stefan240** [comment]
Awesome no problem. I had often similar issues when testing a friend who was also testing it then suggest this. seems a weird fix but works. I work on improving the docs right now where this fix will be added to a Troubleshooting-Readme-File. Really all I am wainting on is for this waiting on an answer of the repo-owner on my PR

**Comment by Synthiss** [comment]
![Screenshot (5)](https://user-images.githubusercontent.com/97289178/148542902-f42e1137-8950-48be-b1f6-e21c537aa3d1.png)
One more thing. After selecting the pre made character I get this error.

**Comment by Stefan240** [comment]
Likely #4 I would guess. Try the solution listed there.

**Comment by arseneangel** [comment]
> Well the 404 indicates the launcher cannot reach the server. Two things to check here at first:
> 
> Make sure the server is actually running and not erroring out after being started. Powershell (or cmd) will show you a log after the server component is started with `go run .`
> 
> If you get something strange while starting the server you may copy the output here in a codeblock, just make sure to omit any personal info.
> 
> After that make sure you edited your hosts-file correctly according to here: https://github.com/ricochhet/Erupe#step-4-edit-your-hosts-file
> 
> Also did you run any other MHF-Server before setting up this one?


So we did do those and checked to make sure everything is right and it was, and i have never ran a mhf server before this is my first one

**Comment by Stefan240** [comment]
> @Synthiss Is this the first MHF-Server you ever connected to on this machine? You may try this since sometimes the launcher can act funny:
> 
> Sometimes the Monster Hunter Frontier Client can act funky when testing things and reinstalling or changing the server you play on. As a first step when troubleshooting things in the MHF-Client it can help to delete _Internet Explorers_ (not Edge) Cache. This is pretty easy.
> 
>     * Open up Internet Explorer
> 
>     * Click on the little cogwheel on the top right
> 
>     * Select Safety -> Delete browsing history
> 
>     * In the now opening window check all the boxes and hit "Delete".
> 
> 
> After that restart the launcher

@arseneangel maybe this one also can help you out as well? Trying it won't hurt surely. The launcher sometimes can be weird

**Comment by arseneangel** [comment]
i will try that and report back after

**Comment by arseneangel** [comment]
so the 404 stopped and now im in a screen with all japanese writing with one button that just closes the window

**Comment by Stefan240** [comment]
> ![Screenshot (5)](https://user-images.githubusercontent.com/97289178/148542902-f42e1137-8950-48be-b1f6-e21c537aa3d1.png) One more thing. After selecting the pre made character I get this error.

Something along this?

**Comment by arseneangel** [comment]
nope this 
![Screenshot 2022-01-07 145157](https://user-images.githubusercontent.com/97197315/148599301-e43899d2-e87a-40ab-a5a7-65036cce96c3.png)

**Comment by Stefan240** [comment]
![grafik](https://user-images.githubusercontent.com/93614125/148599622-407e083b-932c-42e1-8e18-be5b09df8db2.png)
Just went and put it through OCR and see what we get. Anything in the servers logs? also is the server running properly?

**Comment by arseneangel** [comment]
we are looking into that currently

**Comment by Stefan240** [comment]
If you have any output in the shell that may help solve this just post the code. Be sure to omit personal info if there is some beforehand

**Comment by arseneangel** [comment]
i think i got it open, what is the username?

**Comment by arseneangel** [comment]
nvm the last thing i got it but is this right?
![image](https://user-images.githubusercontent.com/97197315/148602218-f4c61b0b-0828-4cc1-84a7-29f07148bfe0.png)

**Comment by Stefan240** [comment]
Please read the guide given: https://github.com/ricochhet/Erupe#step-5-run-the-client-patcher See point 3 here. Anything will suffice. Also the character screen is fine like that. Please be aware that the High Grade setting crashes the server atm it seems. Go into the config (on that page you posted) and uncheck the High Grade option to atleast get ingame. 

Also I would love to hear what was the culprit resulting in this 

> ![grafik](https://user-images.githubusercontent.com/93614125/148599622-407e083b-932c-42e1-8e18-be5b09df8db2.png) Just went and put it through OCR and see what we get. Anything in the servers logs? also is the server running properly?

and what solved the issue for you

**Comment by arseneangel** [comment]
is it supposed to start with previous character data? (i was closing thew go. run window or whatever it is)

**Comment by Stefan240** [comment]
You will create a new character anyway after starting the game. So no thats no problem. Also make sure to keep the server running next time ;)

**Comment by arseneangel** [comment]
> You will create a new character anyway after starting the game. So no thats no problem. Also make sure to keep the server running next time ;)

so just change the settings and hopfulley i should be okay? gosh this was such a chore to do lol i hope it will be worth it

**Comment by Stefan240** [comment]
I mean you have to understand that a lot of stuff is not working with this private server and won't for a long time. So don't get your hopes to high. 

Anyway when you come ingame all the issue should be resolved I guess? Think about closing the issue when this is the case ;)

**Comment by arseneangel** [comment]
the game opened and im in character creation, but there is absolutely no sound

**Comment by Stefan240** [comment]
there is a new issue in #6 so it would be best continue this discussion there and close this issue ^^
